### ISSUE TITLE
Feedback Dismiss Button Is Read Incorrectly When Using TalkAloud.

### ISSUE THREAD

**Issue by JB184351** [woocommerce/woocommerce-android#7403]
**Describe the bug**
On the app Feedback screen the dismiss button, when using TalkAloud is read as "Navigate Up Button", I feel this should be read as "Close" or "Dismiss".

**To Reproduce**
1. Go to Menu
2. Click on Settings
3. Scroll down to 'Send Feedback'
4. Toggle On TalkBack
5. Select the Dismiss Button
6. The app reads it back as `Navigate Up Button`

**Screenshots**
<img src="https://user-images.githubusercontent.com/36495451/190708331-054f8ecd-5ef8-4088-a3e9-8949540756ff.jpg" width="300"/>

**Isolating the problem (mark completed items with an [x]):**
- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: SamSung Galaxy A51
- Android version: 12, One UI 4.1 (not sure if this is helpful or not)
- WooCommerce Android version: 10.1
